### ISSUE TITLE
Email: ITI Comments on First Public Working Draft of W3C Accessibility Guidelines (WCAG) 3.0 (17 - Conformance Model)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#459]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0012.html): 
from [PDF](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/att-0012/ITI_Comments_on_WCAG_3.0_First_Public_Working_Draft_20210224_1_.pdf)

7.	8.0 Conformance model:  We seek feedback on the conformance approach as a whole.

Regarding the current state of the conformance model, there is insufficient detail to fully evaluate it. The general concept of defining thresholds for adjectival ratings and scores is good conceptually, but several questions remain:

1. How are the thresholds for the adjectival ratings determined?
2. Will the thresholds be stable over time or will they change to become more strict or possible to become more lenient as changes in technology provide improvements to support end users (or will those be reflected in the outcomes themselves?)
3. The threshold of 3.5 for Bronze conformance represents another potentially adjustable value that impacts conformance. The draft indicates that the intent is to align bronze conformance with a 90% “score” but it is very difficult to see how that alignment is achieved when looking at the ratings for the outcomes that are currently presented. We believe that the general approach is workable, but wonder why the additional 3.5 factor is needed – couldn’t the outcome ratings be adjusted to eliminate the need for this multiplier? Can the group provide a rationale for why the multiplier is needed?

It would help evaluate and understand the scoring scheme if there was a sample scoring sheet provided. It is very hard to determine the difficulty of conformance scoring across different functional groupings and in aggregate. Understanding whether the model is workable without the ability to view it in this context is a substantial challenge.

Right now, the only level specified is Bronze, as the holistic tests are not defined for Silver and Gold levels. While we cannot comment on the holistic tests, a key concern with these tests is the level of effort required to conduct the testing and how objectively the requirements can be assessed.

We do note and appreciate the Editor’s note on addressing representative sampling in a future draft. This is a very important topic, and relates to our point about flexibility for conformance.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
